common/src/test/java/org/bonitasoft/console/common/server/preferences/properties/SimplePropertiesTest.java

```diff
@@ -50,6 +50,16 @@ public void should_getProperty_return_the_right_custom_permissions() throws Exce
     }
 
     @Test
+    public void should_getProperty_return_the_right_custom_permissions_with_special_characters() throws Exception {
```

Comment:
This is not an English sentence ! Lets read that:
> SimpleProperties should getProperty return the right custom permissions with special characters

hum......
should be `should_return_the_right_custom_permissions_with_special_characters`